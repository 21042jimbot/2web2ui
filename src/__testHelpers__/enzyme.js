export const findButton = (wrapper, buttonLabel) =>
  wrapper.findWhere(
    (node) => node.type() === 'button' && new RegExp(buttonLabel).test(node.text())
  );
