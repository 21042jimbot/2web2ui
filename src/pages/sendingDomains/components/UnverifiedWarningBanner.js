import React from 'react';
import { Banner } from 'src/components/matchbox';
import { LINKS } from 'src/constants';

const UnverifiedWarningBanner = () => {
  const action = {
    content: 'Learn more about review requirements',
    to: LINKS.SENDING_REQS,
    external: true,
  };

  return (
    <Banner
      status="warning"
      title="Unverified sending domains will be removed two weeks after creation."
      action={action}
      my={10}
    >
      <p>All domains will also need a clear web presence to pass verification.</p>
    </Banner>
  );
};

export default UnverifiedWarningBanner;
