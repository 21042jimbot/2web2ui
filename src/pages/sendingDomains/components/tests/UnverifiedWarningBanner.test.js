import { shallow } from 'enzyme';
import React from 'react';

import UnverifiedWarningBanner from '../UnverifiedWarningBanner';

describe('Unverified Domain Warning Banner', () => {
  it('renders correctly', () => {
    expect(shallow(<UnverifiedWarningBanner/>)).toMatchSnapshot();
  })
});
