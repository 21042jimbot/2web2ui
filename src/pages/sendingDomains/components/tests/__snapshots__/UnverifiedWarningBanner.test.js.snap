// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Unverified Domain Warning Banner renders correctly 1`] = `
<Banner
  action={
    Object {
      "content": "Learn more about review requirements",
      "external": true,
      "to": "https://www.sparkpost.com/docs/getting-started/requirements-for-sending-domains",
    }
  }
  status="warning"
  title="Unverified sending domains will be removed two weeks after creation."
>
  <p>
    All domains will also need a clear web presence to pass verification.
  </p>
</Banner>
`;
